### PR TITLE
fixed ui issues

### PR DESCRIPTION
**Pull request by rbdiwash**
1. Fixed left right padding on Homepage and Aboutus page
2. Fixed padding on Header and footer
3. Made buttons consistents
4. Removed unnecessary classes on buttons

Changed according to the review and feedback

### CODE REVIEW

**Review by Aaasisss — COMMENTED** [review]
Some minor suggestions

File: src/modules/home/communityBenefits/communityBenefits.tsx [hunk 1]
```diff
@@ -4,22 +4,24 @@ import { Button } from "@/components/ui/buttons/button";
...
-    <section className="md:flex-col-2 flex flex-col justify-center gap-10 lg:gap-12 mx-10 lg:mx-14 lg:mt-8">
-      <h1 className="text-center text-2xl font-bold lg:text-left">
-        Our Community Benefits
-      </h1>
-      <section className="grid grid-cols-1 gap-12 sm:grid-cols-2 md:grid-cols-3 md:gap-16 lg:grid-cols-4">
-        {communityBenefitsList.map(({ name, imageUrl }) => 
-          <CommunityBenefitCard key={name} name={name} imageUrl={imageUrl} />
-        )}
+    <section className="container mx-auto">
+      <section className="md:flex-col-2 flex flex-col justify-center gap-10 lg:mt-8 lg:gap-12">
```
[the remark this answers is not in the document]
Comment (SajanMG) [reply src/modules/home/communityBenefits/communityBenefits.tsx:12]: It looks good to me @Aaasisss has already mentioned most of the things. And I could see there has been changes in the order of tailwind classes which is minor. All good. Good job @rbdiwash and thank you. 👍

**Comment by SajanMG** [comment]
It looks good to me @Aaasisss has already mentioned most of the things. And I could see there has been changes in the order of tailwind classes which is minor. All good. Good job @rbdiwash and thank you. 👍

**Review by SajanMG — COMMENTED** [review]
Good Job @rbdiwash. after all those changes recommended by @Aaasisss, its all good.
Thanks

**Review by Aaasisss — APPROVED** [review]

**Review by sonam-serchan — APPROVED** [review]
Good work!

**Comment by sonam-serchan** [comment]
Can we please get a tick here? @Aaasisss @SajanMG @20was @boltdynamics ?

My tick didn't work as I made the last commit.

**Review by Aaasisss — APPROVED** [review]

**Pull request merged**